dbt_invoke/internal/_utils.py

```diff
@@ -5,7 +5,6 @@
 import platform
 
 import yaml
-
```

Comment:
Why delete this line? Shouldn't line 6 be deleted instead?

Reply:
It's from https://www.python.org/dev/peps/pep-0008/#imports

```
Imports should be grouped in the following order:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.

You should put a blank line between each group of imports.
```

`yaml` and `dbt` are both 3rd party in this case.

Reply:
Makes sense. I mistakenly thought yaml was a standard library import.